derive_builder/tests/bounds_generation.rs

```diff
@@ -0,0 +1,55 @@
+#[macro_use]
+extern crate derive_builder;
+
+/// A struct that deliberately doesn't implement `Clone`.
+#[derive(Debug, Default, PartialEq, Eq)]
+pub struct Dolor(String);
+
+/// Notice that this type derives Builder without disallowing
+/// `Lorem<Dolor>`.
+#[derive(Debug, Clone, Builder, PartialEq, Eq)]
+#[builder(field(private), setter(into))]
```

Comment:
`field(private)` was needed to squelch the warnings about public fields. You're right that I don't need `setter(into)`; I'll remove that.